easybuild/easyconfigs/f/FORD/FORD-6.1.1-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'PythonBundle'
+
+name = 'FORD'
+version = '6.1.1'
+
+homepage = 'https://github.com/Fortran-FOSS-Programmers/ford/wiki'
+description = 'FORD is an automatic documentation generator for modern Fortran programs'
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+]
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
```

Comment:
@SimonPinches please merge these into the first `dependencies`

Reply:
@verdurin Oops! Not quite sure how I managed that but all fixed now ;-)